### PR TITLE
Document how to modify feature gates

### PR DESCRIPTION
**Pull request by camelid**
How to remove them and how to rename them.

cc <https://github.com/rust-lang/rust/pull/79336#discussion_r528627318>
cc @varkor

### CODE REVIEW

File: src/feature-gates.md
Comment (jyn514) [inline src/feature-gates.md:4]: Could we have "stability in code" link here instead, so all the info is one place?

Comment (camelid) [reply src/feature-gates.md]: Done. I thought of doing that before, but wasn't sure. Good suggestion!

**Review by jyn514 — APPROVED** [review]
LGTM unless you want to wait for varkor to review.

**Comment by camelid** [comment]
I'm fine either way. Worst comes to worse, we can always update the docs :)

I pushed a new commit at the same time you submitted your review though, so you might want to take another look.

**Comment by varkor** [comment]
Looks good to me too 👍

**Pull request merged**
